src/main/php/PDepend/Util/Coverage/CloverReport.php

```diff
@@ -127,10 +127,7 @@ public function getCoverage(AbstractASTArtifact $artifact)
         if (0 === $executed && 1 === $executable && 0 < ($endLine - $startLine)) {
             return 100;
         }
-        if ($executed === 0) {
-            return 0;
-        }
-        return (($executed / $executable) * 100);
+        return $executed / max(1, $executable) * 100;
```

Comment:
To be honest the `max` makes it for me a little bit less readable. Isn't it a problem that the wrong variable is checked?

```suggestion
        if ($executable === 0) {
            return 0;
        }
        return $executed / $executable * 100;
```